course-outline.md

```diff
@@ -0,0 +1,99 @@
+
+### Section 2. Working local with git
+
+Learning Objective: Use command-line git to track local file changes. (*more about git*) \
+Slides: [Section 2 Slides](https://htmlpreview.github.io/?https://github.com/bcgov/ds-intro-to-git/blob/learning-objectives/slides/02-working-local/02-working-local.html)
+
+Topics:
+
+- Default branch naming (inclusive language)
+- What is Bash
+- Bash-specific commands: mkdir, rm -r, cd, ls
+- Git Bash commands: init, commit, add, status, log
+- Directory structure
+- Tracking changes made to a text doc
+- What to do when when -m is omitted.
+
+Course Reference Material:
+
+- [SW Carpentry Episode 3. Creating a Repository (Local)](https://swcarpentry.github.io/git-novice/03-create.html)
+- [SW Carpentry Episode 4. Tracking Changes](https://swcarpentry.github.io/git-novice/04-changes.html)
+- [bcgov-r-geo-workshop Lesson 2](https://github.com/bcgov/bcgov-r-geo-workshop/blob/master/20191105_Day_1_PM_Intro/lessons/02-create.md)
+- [bcgov-r-geo-workshop Lesson 3](https://github.com/bcgov/bcgov-r-geo-workshop/blob/master/20191105_Day_1_PM_Intro/lessons/03-changes.md)
+- [bcgov-r-geo-workshop Lesson 5](https://github.com/bcgov/bcgov-r-geo-workshop/blob/master/20191105_Day_1_PM_Intro/lessons/05-ignore.md)
+
+### Section 3. Remote Code Hosting and GitHub
+
+Learning Objective: What are remote hosting platforms and why do we use them? \
+Slides: [Section 3 Slides](https://htmlpreview.github.io/?https://github.com/bcgov/ds-intro-to-git/blob/learning-objectives/slides/03-remote-hosting/03-remote-hosting.html)
+
+Topics:
+
+- More on secure authentication/login: username/password/2FA to sign in to account, PAT to authenticate
+- Git Bash commands: push, pull
+- Create a remote repository
+- Clone a remote repostitory
+- Push local changes to GitHub.
+- .gitgnore and subdirectories (we don't want ppl pushing data to GitHub)
+- .gitkeep (time permitting) 
+
+Course Reference Material:
+
+- [SW Carpentry Episode 6. Ignoring Things](https://swcarpentry.github.io/git-novice/06-ignore.html)
+- [SW Carpentry Episode 7. Remotes in GitHub](https://swcarpentry.github.io/git-novice/07-github.html)
+- [bcgov-r-geo-workshop Lesson 6](https://github.com/bcgov/bcgov-r-geo-workshop/blob/master/20191105_Day_1_PM_Intro/lessons/06-github.md)
+- [Github: Keeping your Account and Data Secure](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-authentication-to-github)
+
+### Section 4
+
+Learning Objective: Understand how the bcgov framework, policy and related tools help bc government employees share code on GitHub \
+Slides: [Section 4 Slides](https://htmlpreview.github.io/?https://github.com/bcgov/ds-intro-to-git/blob/learning-objectives/slides/04-bcgov-org/04-bcgov-org.html)
+
+Topics:
+
+- what is the bcgov organization framework
+- bcgov policy framework and licensing
+- code-sharing and teams
+- introduction to bcgovr
+- bcgov-c, bcgov enterprise
+
+Course Reference Material:
```

Comment:
I might also include links to useful bcgov materials:

 [Content Approval Checklist](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/blob/master/BC-Open-Source-Development-Employee-Guide/Content-Approval-Checklist.md)

[BC Gov Organizations in GitHub](https://docs.developer.gov.bc.ca/bc-government-organizations-in-github/)

[Policy Framework](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/tree/master)